lib/nexpose/site.rb

```diff
@@ -178,34 +178,6 @@ def discovery_connection_id=(value)
       @discovery_connection_id = value.to_i
     end
 
-    # Adds an asset to this site by host name.
-    #
-    # @param [String] hostname FQDN or DNS-resolvable host name of an asset.
-    def add_host(hostname)
-      @assets << HostName.new(hostname)
-    end
-
-    # Remove an asset to this site by host name.
-    #
-    # @param [String] hostname FQDN or DNS-resolvable host name of an asset.
-    def remove_host(hostname)
-      @assets = assets.reject { |asset| asset == HostName.new(hostname) }
-    end
-
-    # Adds an asset to this site by IP address.
-    #
-    # @param [String] ip IP address of an asset.
-    def add_ip(ip)
-      @assets << IPRange.new(ip)
-    end
-
-    # Remove an asset to this site by IP address.
-    #
-    # @param [String] ip IP address of an asset.
-    def remove_ip(ip)
```

Comment:
`#remove_asset` calls both `#remove_host` and `#remove_ip` unlike add_asset which performs the array concat logic.